static/src/javascripts/projects/common/modules/navigation/edition-picker.js

```diff
@@ -0,0 +1,69 @@
+define([
+    'common/utils/fastdom-promise',
+    'common/utils/$'
+], function (
+    fastdomPromise,
+    $
+) {
+    var dropdown = $('.js-edition-picker-dropdown');
+
+    function enhanceToButton() {
+        var container = $('.js-edition-picker-container');
+        var checkBox = $('.js-enhance-checkbox');
+        var button = $.create('<button>');
+
+        button.addClass('edition-picker__button js-open-edition-picker');
+        button.attr('id', 'edition-picker');
+        button.attr('aria-controls', 'edition-picker__dropdown');
+        button.attr('aria-expanded', 'false');
+
+        fastdomPromise.write(function() {
+            checkBox.remove();
+            container.prepend(button);
+
+            openMenuOnClick();
+        });
+    }
+
+    function openMenuOnClick() {
+        var button = $('.js-open-edition-picker');
+
+        if (button.length > 0) {
+            button[0].addEventListener('click', function() {
+
+                fastdomPromise.write(function() {
+                    if (button.hasClass('open')) {
+                        button.removeClass('open');
+                        button.attr('aria-expanded', 'false');
+                        dropdown.attr('hidden', '');
+                    } else {
+                        button.addClass('open');
+                        button.attr('aria-expanded', 'true');
+                        dropdown.removeAttr('hidden');
+                    }
+                });
+            });
+        }
+    }
+
+    function clickMenuOnEnter() {
+        var label = document.getElementsByClassName('js-on-enter-click')[0];
```

Comment:
Is it possible for an element with this class to *not* be present on the page? If so, this line will fall over (whereas the pattern used on lines 29 - 32 above is more defensive)